test_unstructured_ingest/metrics/all-docs-cct.tsv

```diff
@@ -1,3 +1,3 @@
-filename	connector	cct-accuracy	cct-%missing

```

Comment:
not a problem, added doctype column and (for some reason) switched order of documents. scores unchanged